apps/examples/testcase/ta_tc/ble_manager/itc/itc_blemanager_main.c

```diff
@@ -0,0 +1,865 @@
+/****************************************************************************
+ *
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+#include <ble_manager/ble_manager.h>
+#include <semaphore.h>
+#include <stdbool.h>
+#include <errno.h>
+#include "tc_common.h"
+
+#define LOOP_SIZE 2
```

Comment:
This macro has been used at a few places,
Could you please add a brief description about where all it is used and how different functions/cases use the same number of loop iterations?

Reply:
Sure, done